index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>CV Dzmitry Kulevich</title>
+	<meta charset="utf-8">
+	<link rel="stylesheet" href="cv.css">
+</head>
+<body>
+	<main>
+		<h1>CV Dzmitry Kulevich</h1><hr>
+		<section id="personal-information">
+			<img src="cv-photo.jpg" alt="Photo for CV" width="200" height="230">
+			<h3>Personal Information</h3>
+			<table>
+				<tbody>
+					<tr>
+						<td><span class=bold>Name:</span></td>
+						<td><span class="italic">Dzmitry</span></td>
+					</tr>
+					<tr>
+						<td><span class=bold>Last Name:</span></td>
+						<td><span class="italic">Kulevich</span></td>
+					</tr>
+					<tr>
+						<td><span class=bold>Date of birth:</span></td>
+						<td><span class="italic">20/11/1987</span></td>
+					</tr>
+					<tr>
+						<td><span class=bold>Nationality:</span></td>
+						<td><span class="italic">Belarus</span></td>
+					</tr>
+					<tr>
+						<td><span class=bold>Marital status:</span></td>
+						<td><span class="italic">Married</span></td>
+					</tr>
+				</tbody>
+			</table>
+		</section>
+		<section id="contact-information">
+			<h3>Contact Information</h3>
+			<table>
+				<tbody>
+					<tr>
+						<td><span class=bold>Address:</span></td>
+						<td><span class="italic">21 Alibegova Street, apt. 21, Minsk, Belarus</span></td>
+					</tr>
+					<tr>
+						<td><span class=bold>Phone number:</span></td>
+						<td><span class="italic">+375(29)778-04-41</span></td>
+					</tr>
+					<tr>
+						<td><span class=bold>Email:</span></td>
+						<td><span class="italic">selestrel07@gmail.com</span></td>
```

Comment:
FYI:
you may use ```a``` tag here:
`<a href="mailto:selestrel07@gmail.com">selestrel07@gmail.com</a>`
it will make possible to click the link in any browser (including mobile ones) and open an application to create an email with the predetermined content including subject, body, etc.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Creating_an_email_link